src/invoices/InvoiceDetails/CancellationModal/CancellationModal.js

```diff
@@ -16,9 +16,9 @@ import styles from './CancellationModal.css';
 const CancellationModal = ({
   onCancel,
   onConfirm,
+  setCancellationNote,
```

Comment:
To reflect your changes, hook dependencies arrays must include `setCancellationNote` where it is used.